datafusion/tests/sql.rs

```diff
@@ -1281,6 +1281,60 @@ async fn csv_query_approx_count() -> Result<()> {
     Ok(())
 }
 
+#[tokio::test]
+async fn csv_query_array_agg() -> Result<()> {
+    let mut ctx = ExecutionContext::new();
+    register_aggregate_csv(&mut ctx).await?;
+    let sql =
+        "SELECT array_agg(c13) FROM (SELECT * FROM aggregate_test_100 LIMIT 2) test";
```

Comment:
can you please add an `ORDER BY` to this query - as written it may be non deterministic (the first two values returned are not guaranteed to be in any order):

```suggestion
        "SELECT array_agg(c13) FROM (SELECT * FROM aggregate_test_100 ORDER BY c13 LIMIT 2) test";
```